### ISSUE TITLE
Welcome Message is JS specific

### ISSUE THREAD

**Issue by rlemon** [Zirak/SO-ChatBot#133]
The welcome message isn't useful for anyone but us (in its current form). For other rooms (or all rooms) maybe it should say this:   

    Welcome to the <Room Name> chat! <Room Description>. Please don't ask if you can ask or if anyone's around; just ask your question, and if anyone's free and interested they'll help.

That way it still gets the point a crossed and we can even have it changing with the season. Example: 

    Welcome to the JavaScript chat! Happy Ho Ho Day!  Please don't ask if you can ask or if anyone's around; just ask your question, and if anyone's free and interested they'll help.

Comments?

**Comment by onebree** [comment]
Can confirm that this is still an issue. In the Ruby room:
http://chat.stackoverflow.com/transcript/message/28204436#28204436

Furthermore,  `!!welcome` should be used with a leading `@` with the username.

**Comment by rlemon** [comment]
no, double pings with @ is not going to happen. This has been discussed to death. 

Why this [likely] still hasn't been done.   
1) what should the message say? 
2) Do we include the room description? 
3) how do we link rules in the help message for every room?
4) how does a room update their own message without forking the bot and making a PR?

There is probably more to consider still, but that is a start.

**Comment by onebree** [comment]
@rlemon I did not think of that, with the double pings.

I did not look too much into `welcome.js`, and I am also very weak in JS. But what about something like the following?

```js
var room = document.getElementById("roomname").innerText;
var desc = document.getElementById("roomname").innerHTML;
// some logic to parse desc
// bot.adapter.link();

var message = name + '  Welcome to ' + room + '! ' + desc;
```

**Comment by allquixotic** [comment]
What about just building in a custom welcome message string (or a function that returns a string) into a config.js, with the default of the JS room? 

We've been wanting config.js for various things, including but not limited to the login credentials and the bot's home room to pass to run-headless.js, for a while now. 

NB: This change alone wouldn't enable us to stop forking the bot. We have plenty of other things we do differently, commits we've reverted, commands we've deleted (or undeleted), etc.
